types/variant.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
The VARIANT data type is supported in 15 Runtime and above!!!. In all other Runtimes it should be disabled